File: comms/bulk-messaging/sender-pools/create-sender-pool/create-sender-pool.ts
import { TwilioClient } from "twilio-comms";

async function main() {
    const client = new TwilioClient({
        accountId: "TWILIO_ACCOUNT_SID",
        authToken: "TWILIO_AUTH_TOKEN",
    });
    await client.senderPools.create({
        name: "Sales Leads - APAC",
        tags: {
            "region": "APAC",
        },
    });
}
main();

File: comms/bulk-messaging/sender-pools/create-sender-pool/create-sender-pool.py
from twilio_comms import TwilioComms

client = TwilioComms(
    account_id="TWILIO_ACCOUNT_SID",
    auth_token="TWILIO_AUTH_TOKEN",
)

client.sender_pools.create(
    name="Sales Leads - APAC",
    tags={
        "region": "APAC"
    },
)
